templates/method/drupal-cms.hbr

```diff
@@ -138,10 +154,18 @@
                             {{/required}}
                         </td>
                         <td>
-                            {{#if required}}
-                            <input type="text" class="form-control" data-role="value" value="{{defaultValue}}"/>
+                            {{#if options}}
+                                <select name="{{name}}" data-role="multiple-value">
+                                    {{#options}}
+                                        <option value="{{.}}">{{.}}</option>
```

Comment:
I finally tested this for Content-Type and it works correctly.  There may be other bugs , but DEVSOL-1878 is fixed, please accept and add another issue if there are problems with other data.  I was not able to reproduce other options.